homu/server.py

```diff
@@ -1,7 +1,8 @@
 import hmac
 import json
 import urllib.parse
-from .main import PullReqState, parse_commands, db_query, INTERRUPTED_BY_HOMU_RE, synchronize
+from .main import (PullReqState, parse_commands, db_query,
+                   INTERRUPTED_BY_HOMU_RE, synchronize)
```

Comment:
OK, I just remembered the spot where block style indents fall apart: long function declarations. 

Before: 
```
def parse_commands(body, username, repo_cfg, state, my_username, db, states, *, realtime=False, sha=''):                                  
    global global_cfg                                                           
    state_changed = False   
```

Just sticking in a line break:

```
def parse_commands(body, username, repo_cfg, state, my_username, db, states,    
                   *, realtime=False, sha=''):                                  
    global global_cfg                                                           
    state_changed = False   
```

Full block-style indent needs the end paren with no indent, which looks really funny to me: 

```
def parse_commands(                                                             
    body,                                                                       
    username,                                                                   
    repo_cfg,                                                                   
    state,                                                                      
    my_username,                                                                
    db,                                                                         
    states,                                                                     
    *,                                                                          
    realtime=False,                                                             
    sha=''                                                                      
):                                                                              
    global global_cfg                                                           
    state_changed = False    
```